locale/pt-br/texts.po

```diff
@@ -21,25 +30,337 @@ msgstr ""
+msgid "Start the Wallet"
+msgstr "Iniciar a Wallet"
+
+#: src/screens/CreateToken.js:70
+msgid "You don't have any hathor tokens available to create your token."
+msgstr "Você não tem nenhum token Hathor disponível para criar o seu token."
+
+#: src/screens/CreateToken.js:85
+msgid "All fields must be filled."
+msgstr "Todos os campos devem ser preenchidos."
+
+#: src/screens/CreateToken.js:123 src/screens/CreateTokenConfirm.js:84
+msgid "Enter your 6-digit pin to create your token"
+msgstr "Entre com seu PIN de 6 digitos para criar o seu token"
+
+#: src/screens/CreateToken.js:124 src/screens/CreateTokenConfirm.js:85
+msgid "Authorize token creation"
+msgstr "Autorizar criação de token"
+
+#: src/screens/CreateToken.js:140
+msgid "CREATE NEW TOKEN"
+msgstr "CRIAR UM NOVO TOKEN"
+
+#: src/screens/CreateToken.js:146 src/screens/CreateTokenName.js:51
+msgid "Token Name"
+msgstr "Nome do Token"
+
+#: src/screens/CreateToken.js:153
+msgid "Token Symbol"
+msgstr ""
+
+#: src/screens/CreateToken.js:161 src/screens/PaymentRequestDetail.js:108
+msgid "Amount"
+msgstr "Quantidade"
+
+#: src/screens/CreateToken.js:172
+msgid "Create New Token"
+msgstr "Criar Novo Token"
+
+#: src/screens/CreateTokenAmount.js:109 src/screens/CreateTokenConfirm.js:126
+#: src/screens/CreateTokenDepositNotice.js:39 src/screens/CreateTokenName.js:44
+#: src/screens/CreateTokenSymbol.js:63
+msgid "CREATE TOKEN"
+msgstr "CRIAR TOKEN"
+
+#: src/screens/CreateTokenAmount.js:117
+#, javascript-format
+msgid "Amount of ${ this.name } (${ this.symbol })"
+msgstr "Quantidade de ${ this.name } (${ this.symbol })"
+
+#: src/screens/CreateTokenAmount.js:129
+msgid "Deposit:"
+msgstr "Depósito:"
+
+#: src/screens/CreateTokenAmount.js:133
+#, javascript-format
+msgid "You have ${ amountAvailableText } HTR available"
+msgstr "Você tem ${ amountAvailableText } HTR disponíveis"
+
+#: src/screens/CreateTokenAmount.js:138 src/screens/CreateTokenName.js:65
+#: src/screens/CreateTokenSymbol.js:85 src/screens/InitWallet.js:208
+#: src/screens/InitWallet.js:328 src/screens/SendAddressInput.js:66
+#: src/screens/SendAmountInput.js:159
+msgid "Next"
+msgstr "Próximo"
+
+#.  show loading modal
+#.  eslint-disable-next-line react/jsx-indent
+#: src/screens/CreateTokenConfirm.js:66
+msgid "Creating token"
+msgstr "Criando um token"
+
+#: src/screens/CreateTokenConfirm.js:135
+#, javascript-format
+msgid "Amount of ${ this.name }"
+msgstr "Quantidade de ${ this.name }"
+
+#: src/screens/CreateTokenConfirm.js:143
+msgid "Token name"
+msgstr "Nome do token"
+
+#: src/screens/CreateTokenConfirm.js:149 src/screens/CreateTokenSymbol.js:70
+msgid "Token symbol"
+msgstr "Símbolo do token"
+
+#: src/screens/CreateTokenConfirm.js:155
+msgid "Deposit"
+msgstr "Depósito"
+
+#: src/screens/CreateTokenConfirm.js:162
+msgid "Create token"
+msgstr "Criar um token"
+
+#: src/screens/CreateTokenDepositNotice.js:45
+msgid "When creating new tokens, a "
+msgstr "Quando um novo token é criado, um "
```

Comment:
This type of text should not be broken up into separate translations. I think it should be possible to interpolate text and component with ttag to allow this type of text to not be broken up.

Reply:
Fixed.